### PR TITLE
Update the tutorial build scripts with 1.9.20 APIs

### PR DESCRIPTION
**Pull request by abdulowork**
This PR updates [this tutorial](https://kotlinlang.org/docs/multiplatform-mobile-ktor-sqldelight.html) with 1.9.20 build script APIs. The corresponding updates in the sample:
* [Initial state](https://github.com/kotlin-hands-on/kmm-networking-and-data-storage/pull/32)
* [Final state](https://github.com/kotlin-hands-on/kmm-networking-and-data-storage/pull/31)

### CODE REVIEW

File: docs/topics/multiplatform-mobile/multiplatform-mobile-ktor-sqldelight.md [hunk 1]
```diff
@@ -117,32 +115,12 @@ Both the `kotlinx.serialization` and SQLDelight libraries also require additiona
...
-            classpath("com.squareup.sqldelight:gradle-plugin:%sqlDelightVersion%")
-        }
-    }
-    ```
-
-4. Finally, define the SQLDelight version in the `gradle.properties` file in the project _root directory_ to ensure that
-   the SQLDelight versions of the plugin and the libraries are the same:
-
-    ```none
-    sqlDelightVersion=%sqlDelightVersion%
```
Comment (abdulowork) [inline docs/topics/multiplatform-mobile/multiplatform-mobile-ktor-sqldelight.md:142]: I couldn't find a reference to this property anywhere. Could it have been left by accident from an earlier tutorial revision or did I miss something?

File: docs/topics/multiplatform-mobile/multiplatform-mobile-ktor-sqldelight.md [hunk 2]
```diff
@@ -117,32 +115,12 @@ Both the `kotlinx.serialization` and SQLDelight libraries also require additiona
...
 
-3. Now go to the `build.gradle.kts` file in the project _root directory_ and specify the classpath for the plugin in the
-   build system dependencies:
-
-    ```kotlin
-    buildscript {
-        
-        dependencies {
-            // ...
-            classpath("com.squareup.sqldelight:gradle-plugin:%sqlDelightVersion%")
```
Comment (abdulowork) [inline docs/topics/multiplatform-mobile/multiplatform-mobile-ktor-sqldelight.md:133]: Is there any reason to specify buildscript dependencies explicitly? I think the explicit version [in the plugins](https://github.com/JetBrains/kotlin-web-site/pull/3824/files#diff-6d95cf38840a09a14676b740a25fa2dde290007c571510ff620ef14bb077cb9aR119) block achieves the same goal

File: docs/topics/multiplatform-mobile/multiplatform-mobile-ktor-sqldelight.md (hunk 1 above)
Comment (danil-pavlov) [reply docs/topics/multiplatform-mobile/multiplatform-mobile-ktor-sqldelight.md:142]: Yes, I guess it's something old. If it's not used in the sample project, we can remove it

**Review by danil-pavlov — APPROVED** [review]
Since these changes concern 1.9.20, we're gonna merge them along the 1.9.20 docs update. could you rebase your branch to 1-9-20-docs?

**Pull request merged**